src/detection/auto_calibration_4q.py

```
@@ -0,0 +1,67 @@
+    pupil_center = None
+    max_area = 0
+
+    for cnt in contours:
+        area = cv2.contourArea(cnt)
+        if area < 50 or area > 2000:  # 너무 작거나 큰 노이즈 제외
+            continue
+
+        (x, y), radius = cv2.minEnclosingCircle(cnt)
+        circularity = 4*np.pi*area/(cv2.arcLength(cnt, True)**2)
+        if 0.4 < circularity < 1.2:  # 원형일 경우만
+            if area > max_area:  # 가장 큰 원 선택
```

Ruff (0.12.2)

Use a single `if` statement instead of nested `if` statements

(SIM102)